CHANGELOG.md

```diff
@@ -5,11 +5,12 @@ FEATURES:
 * **New Resource:** `vcd_lb_service_monitor`  Load Balancer Service Monitor - [GH-256]
 * **New Data Source:** `vcd_lb_service_monitor` Load Balancer Service Monitor  - [GH-256]
 * **New Resource:** `vcd_edgegateway` creates and deletes edge gateways - [GH-262]
-* **New build command** `make test-env-build` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
+* **New build commands** `make test-env-init` and `make test-env-apply` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
```

Comment:
Please move this section to the bottom of the list, so it stands out on its own (as opposed to in the group of LB additions).

Reply:
moved